.github/workflows/test.yml

```
@@ -0,0 +1,50 @@
+name: Test
+# Test the python packages that are part of the toolchain
+
+on:
+    push:
+        branches: [ "main" ]
+    pull_request:
+        branches: [ "main" ]
+    
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                ros-distro: [ "humble" ]
+        steps:
+            # Checkout the repository
+            - name: Checkout repository
+              uses: actions/checkout@v2
+            # Set up ROS
+            - name: Set up ROS
+              uses: ros-tooling/setup-ros@v0.7
+              with:
+                required-ros-distributions: ${{ matrix.ros-distro }}
+            # Get smc_storm for testing
+            - name: Get smc_storm
+              id: get_smc_storm
+              run: |
+                wget https://github.com/convince-project/smc_storm/releases/download/0.0.1/smc_storm_executable.tar.gz
```

Why not using latest here?

---

I thought it gets more predictable results when fixing the version.

---

Hmm, I think you have a point here. What about making this more explicit then, adding an entry for the smc_storm version and using it in the wget command afterwards?
The easiest way would be doing everything in bash:
```suggestion
                SMC_STORM_VERSION=0.0.1
                wget https://github.com/convince-project/smc_storm/releases/download/${SMC_STORM_VERSION}/smc_storm_executable.tar.gz
```